adapters/mobilefuse/mobilefuse.go

```diff
@@ -0,0 +1,193 @@
+package mobilefuse
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/golang/glog"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"strconv"
+	"text/template"
+)
+
+type MobilefuseAdapter struct {
+	EndpointTemplate template.Template
+}
+
+func NewMobilefuseBidder(endpointTemplate string) adapters.Bidder {
+	parsedTemplate, errs := template.New("endpointTemplate").Parse(endpointTemplate)
```

Comment:
Nitpick: Please change `errs` to `err` to make it clear that the result is a single error and not an error slice.